admin/data/sql/migrations/028.do.index-pupil-attendance.sql

```diff
@@ -0,0 +1,2 @@
+-- a pupil can only have one entry in the pupilAttendance table
+CREATE UNIQUE INDEX pupilAttendance_pupil_id_uindex ON mtc.mtc_admin.pupilAttendance (pupil_id);
```

Comment:
might want to update your branch, this migration number is already used

Reply:
Good spot!  Now renamed.